src/components/add-form/add-form.test.js

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
+import { useContext } from 'react';
+import { AddForm } from './add-form';
+
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useContext: jest.fn(),
+}));
+
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useDispatch: jest.fn(),
+}));
+
+jest.mock('../../hooks/use-get-params/use-get-params', () => ({
+  useGetParams: jest.fn(() => [{}]),
+}));
+
+describe('Add form', () => {
+  beforeEach(() => {
+    useContext.mockImplementationOnce(() => ({
```

Comment:
Since you have one test, `beforeEach` doesn't make much sense.

Reply:
Removed `beforeEach`.